dev/com.ibm.websphere.appserver.features/visibility/auto/com0.ibm.websphere.appserver.jsf.beanValidation-2.3.feature

```diff
@@ -0,0 +1,10 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
watch out, you have a typo in the file name

Reply:
oh, is the `0` somehow intentional? i see the other auto features have similar?

Reply:
however, new features should be named soemthing like `io.openliberty.jsf2.3-beanValidation2.0`

Reply:
Removed these files since they were included by accident. New feature is named `io.openliberty.faces.beanValidation-3.0.feature`